### PR TITLE
Update Starburst Presto to 323e

### PR DESCRIPTION
**Pull request by findepi**
Relates to https://github.com/GoogleCloudPlatform/dataproc-initialization-actions/pull/703
cc @ChethanUK

### CODE REVIEW

File: starburst-presto/presto.sh [hunk 1]
```diff
@@ -22,8 +22,8 @@ readonly ROLE="$(/usr/share/google/get_metadata_value attributes/dataproc-role)"
 readonly PRESTO_MASTER_FQDN="$(/usr/share/google/get_metadata_value attributes/dataproc-master)"
 readonly WORKER_COUNT=$(/usr/share/google/get_metadata_value attributes/dataproc-worker-count)
 readonly PRESTO_BASE_URL=https://storage.googleapis.com/starburstdata/presto
-readonly PRESTO_MAJOR_VERSION="312"
-readonly PRESTO_VERSION="${PRESTO_MAJOR_VERSION}-e.1"
+readonly PRESTO_MAJOR_VERSION="323"
```
Comment (ChethanUK) [inline starburst-presto/presto.sh]:
```
wget https://storage.googleapis.com/starburstdata/presto/323e/323-e.3/presto-server-323-e.3.tar.gz
2020-01-11 17:49:50 ERROR 403: Forbidden.
```
Hey, we talked before right; uploading `presto-server-323-e.3.tar.gz` to GCS is not in the pipeline of the Starburstdata's release cycle...

Comment (findepi) [reply starburst-presto/presto.sh]: Should be accessible now.

Comment (ChethanUK) [reply starburst-presto/presto.sh]: Awesome :)

Comment (findepi) [reply starburst-presto/presto.sh]: @ChethanUK thanks for checking me!

**Pull request merged**